src/modules/expeditions/components/ExpeditionsModal/ExpeditionsModal.tsx

```diff
@@ -0,0 +1,71 @@
+import { useState } from 'react'
+
+import { ReactComponent as SwaprLogo } from '../../../../assets/images/swapr-logo.svg'
+import { AutoColumn } from '../../../../components/Column'
+import { FallbackLoader } from '../../../../components/Loader/FallbackLoader'
+import Modal from '../../../../components/Modal'
+import Row from '../../../../components/Row'
+import { useShowExpeditionsPopup } from '../../../../state/application/hooks'
+import { CloseIcon, TYPE } from '../../../../theme'
+import { ExpeditionsFragmentsBalance } from '../../components/ExpeditionsFragments'
+import { ExpeditionsTabBar, ExpeditionsTabs } from '../../components/ExpeditionsTabBar'
+import { useExpeditions } from '../../contexts/ExpeditionsContext'
+import { ScrollableAutoColumn, StyledExternalLink } from '../shared'
+import { ContentWrapper, ExpeditionsLogo } from './ExpeditionsModal.styled'
+import { ExpeditionsRewards } from './ExpeditionsRewards'
+import { ExpeditionsTasks } from './ExpeditionsTasks'
+export interface ExpeditionsModalProps {
```

Comment:
would add a new line here.

```suggestion

export interface ExpeditionsModalProps {
```